rest/data/task_test.go

```diff
@@ -81,6 +82,25 @@ func (s *TaskConnectorFetchByIdSuite) TestFindByIdAndExecution() {
 	}
 }
 
+func (s *TaskConnectorFetchByIdSuite) TestFindByVersion() {
```

Comment:
Can this test more cases? (A case where annotation exists but Issues is the empty list, Issues is nil,  no annotation exists, etc)

Reply:
added all except for issues is nil. I don't know if that would ever be the case.

Reply:
Issues is an array, not a pointer.

Reply:
I'm adding a test for when it's empty. Please hold of reviewing.

Reply:
done